pkg/runtime/openshell/gateway.go

```
@@ -39,6 +39,8 @@ const (
 	gatewayURL               = "http://127.0.0.1:" + gatewayPort
 )
 
+var supervisorImage = "ghcr.io/nvidia/openshell/supervisor:" + strings.TrimPrefix(DefaultVersion, "v")
```

golangci-lint (2.12.1)

[error] var supervisorImage is unused

(unused)